app/controllers/api/v0/comments_controller.rb

```diff
@@ -9,7 +9,7 @@ class CommentsController < ApiController
       private_constant :ATTRIBUTES_FOR_SERIALIZATION
 
       def index
-        article = Article.find(params[:a_id])
+        article = params[:a_id] ? Article.find(params[:a_id]) : PodcastEpisode.find(params[:p_id])
```

Comment:
Small nitpick: can we rename the variable to `commentable`?

Reply:
Hi @rhymes, I have changed variable name to commentable and tried to add a test, but I am not sure if it is as in depth as you would like (I have never written one before, so mostly just copied what was with article) only I don't know if it should have all the same checks for comments as article does, e.g child comments, grandchild comments, etc... I assume since the search by article id and search by podcast ep id use the same code it doesn't matter practically speaking, but yes with respect to the test I am not sure